view/frontend/web/js/index.vue

```diff
@@ -7,6 +7,8 @@
   </div>
 </template>
 
+<style src="./assets/vue-multiselect.min.css"></style>
+
```

Comment:
Always add styles on the bottom of SFC

Reply:
Change it to `<style src="vue-multiselect/dist/vue-multiselect.min.css">`

Reply:
moved and changed